ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -529,11 +529,11 @@ func (r *RayJobReconciler) createNewK8sJob(ctx context.Context, rayJobInstance *
 	// Create the Kubernetes Job
 	if err := r.Client.Create(ctx, job); err != nil {
 		logger.Error(err, "Failed to create new Kubernetes Job")
-		r.Recorder.Eventf(rayJobInstance, corev1.EventTypeWarning, "k8sJobCreationFailed", "Failed to create new Kubernetes Job %s: %v", job.Name, err)
+		r.Recorder.Eventf(rayJobInstance, corev1.EventTypeWarning, string(utils.FailedToCreateRayJobSubmitter), "Failed to create new Kubernetes Job %s: %v", job.Name, err)
 		return err
 	}
 	logger.Info("Kubernetes Job created", "RayJob", rayJobInstance.Name, "Kubernetes Job", job.Name)
```

Comment:
Change the log message to make the wording of creating a K8s resource consistent with others in `raycluster_controller.go` ([example](https://github.com/ray-project/kuberay/blob/8f061977b41c24b7eb3ea83f35758cbe723f63ff/ray-operator/controllers/ray/raycluster_controller.go#L1425)).

```suggestion
	logger.Info("Created submitter Kubernetes Job for RayJob", "name", job.Name)
```